src/app/[locale]/page.tsx

```diff
@@ -1,5 +1,10 @@
 "use client";
 
+/**
+ * @dev temporary removing some logs in client.
+ */
+require("../../../tools/removeLogs");
```

Comment:
You can't use alias? :)

Reply:
if you do 
```suggestion
import "@/tools/removeLogs"
```
This don't works?

Reply:
Ok when doing that I had an error saying that the file wasn't a module so I just added `module.exports = {}` and it worked. Cool, thanks for the heads up @mcayuelas-ledger !